lib/pf/dal/_action.pm

```diff
@@ -0,0 +1,181 @@
+package pf::dal::_action;
+
+=head1 NAME
+
+pf::dal::_action -
```

Comment:
I think you might need to adjust the template so that a meaningful name and description show up